api/src/org/apache/cloudstack/api/command/admin/guest/ListGuestOsMappingCmd.java

```diff
@@ -44,10 +44,10 @@
     /////////////////////////////////////////////////////
 
     @Parameter(name = ApiConstants.ID, type = CommandType.UUID, entityType = GuestOsMappingResponse.class, required = false, description = "list mapping by its UUID")
-    private Long id;
+    private String id;
```

Comment:
@bhaisaab Thanks for pointing out! Will look into it. @remibergsma Please hold on merging.